NEWS.rst

```diff
@@ -2,6 +2,56 @@
 NEWS
 ******
 
+Version 1.0-rc4
+===============
+
+:date: 2018-01-18
+:commit: nil
+
+Changes:
```

Comment:
add a newline, otherwise the bullet points don't work

Reply:
fixed